src/app/models/field/attachmentField.ts

```diff
@@ -30,7 +30,7 @@ const createAttachmentFieldSchema = () => {
   ) {
     const { webhook, responseMode } = this.parent()
 
-    if (responseMode === ResponseMode.Encrypt && webhook.url) {
+    if (responseMode === ResponseMode.Encrypt && webhook?.url) {
```

Comment:
were we at risk of a null pointer exception here?

Reply:
No, if it's encrypt mode it will have a webhooks object.